product_docs/docs/pgd/5/cli/discover_connections.mdx

```diff
@@ -51,33 +55,41 @@ instances:
 ...
 ```
 
-The connection string for this cluster would be:
+The connection string for this cluster is:
 
 ```
 "host=192.168.100.2 port=6432 dbname=bdrdb user=enterprisedb sslmode=require"
 ```
 
 !!! Note Host name versus IP address
-In our example, we use the IP address because the configuration is from a Docker TPA install with no name resolution available. Generally, you should be able to use the host name as configured.
+The example uses the IP address because the configuration is from a Docker TPA install with no name resolution available. Generally, you can use the host name as configured.
 !!!
 
 ### For a BigAnimal distributed high-availability cluster
 
-1. Log into the [BigAnimal Clusters](https://portal.biganimal.com/clusters) view.
+1. Log in to the [BigAnimal clusters](https://portal.biganimal.com/clusters) view.
 1. Select your cluster.
```

Comment:
is there a way for them to tell Distributed HA clusters vs Primary/Standby HA clusters? if so, we tell them how to ensure they are choosing one of the clusters that's valid for the CLI

Reply:
```suggestion
1. In the filter, set the Cluster Type to "Distributed High Availability" to only show clusters which work with PGD CLI.
1. Select your cluster.
```